client/src/document/on-github.tsx

```diff
@@ -53,7 +52,7 @@ function EditOnGitHubLink({ doc }: { doc: Doc }) {
       target="_blank"
       rel="noopener noreferrer"
     >
-      Edit on <b>GitHub</b>
+      Edit it on <b>GitHub</b>
```

Comment:
How about being even more explicit?

```suggestion
      Edit this page on <b>GitHub</b>
```

Reply:
I think the pronoun 'it' is not ambiguous here.

Reply:
Paging @bsmth to get a third opinion :)

Reply:
I like both. I agree "it" is not ambiguous given we just ask if you found a problem with "this page".

Reply:
The header contains two nouns, "the problem" and "this page", so "it" could refer to both (I guess?). The other three items refer to "the source/content/problem", so maybe "Edit the content/page" would be more consistent in this context.

PS: I don't think "Edit on GitHub" was ambiguous in the first place, but the pronoun "it" still tends to confuse me here.

Reply:
OK I'm leaning more to "Edit this page" now 😄

Reply:
Let's go for whatever the two of you agree on. 😉

Reply:
I am okay going with "Edit the page" to avoid repeating "this". How about the following (more in line with Brian's suggestions):

```
Found a problem with this page?
- Edit the page.
- Report a content issue.
- Learn how to contribute.
```

Reply:
@dipikabh Like this?
<img width="783" alt="image" src="https://user-images.githubusercontent.com/495429/209407566-e327e9cf-b91a-462f-9367-52604edc26a2.png">

I'm not sure about the trailing full stops. 🤔

Reply:
Yeah, even though they are grammatically correct, the periods at the end are adding to the clutter (more apparent in the screenshot you've posted). I like @bsmth's suggestion of dropping the "View" bit.